src/lints/bool_comparison.rs

```diff
@@ -60,6 +60,10 @@ impl Lint for BoolComparison {
     fn fix(&self, db: &dyn SemanticGroup, node: SyntaxNode) -> Option<InternalFix> {
         fix_bool_comparison(db.upcast(), node)
     }
+
+    fn fix_message(&self) -> &'static str {
+        "Simplify to direct boolean check"
```

Comment:
As it's the message that will be specifically be shown to the end user, i would recommend to use the "." at the end of every fix message. This comment relates to all other fix messages